drivers/tmp00x/tmp00x.c

```diff
@@ -202,26 +204,26 @@ int tmp00x_read_temperature(const tmp00x_t *dev, int16_t *ta, int16_t *to)
 {
 
     uint16_t drdy;
-#if (!TMP00X_USE_RAW_VALUES)
+#if (!IS_ACTIVE(CONFIG_TMP00X_USE_RAW_VALUES))
```

Comment:
I think these conditionals can be C `if`.

Reply:
I did try it but it returned error.

Reply:
Probably you need to also change the conditional include of `xtimer.h`

Reply:
Done. Updated and tested.